src/coreclr/tools/Common/TypeSystem/Common/Utilities/CustomAttributeTypeNameParser.cs

```diff
@@ -179,10 +179,10 @@ private TypeDesc GetSimpleTypeFromModule(TypeName typeName, ModuleDesc module)
                     TypeDesc type = GetSimpleTypeFromModule(typeName.DeclaringType, module);
                     if (type == null)
                         return null;
-                    return ((MetadataType)type).GetNestedType(TypeNameHelpers.Unescape(typeName.Name));
+                    return ((MetadataType)type).GetNestedType(TypeName.Unescape(typeName.Name));
                 }
 
-                string fullName = TypeNameHelpers.Unescape(typeName.FullName);
+                string fullName = TypeName.Unescape(typeName.FullName);
```

Comment:
`fullName` is only needed in the `if` block below. Move this inside the `if` block.